index.js

```diff
@@ -116,6 +115,14 @@ AFRAME.registerComponent('controller-cursor', {
   },
 
   /**
+   * Remove mesh.
+   */
+  remove: function () {
+    var cursorEl = this.el;
+    cursorEl.removeObject3D('cursormesh');
```

Comment:
> it looks like pause is called by a-frame itself prior to remove

Yes you're right, my mistake!

> I have not explicitly tested to see if the event listeners are still running

I'm going to trust that they are removed :smile: